libc/src/__support/macros/properties/os.h

```diff
@@ -37,4 +37,8 @@
 #endif
 #endif
 
+#if (defined(__Fuchsia__))
```

Comment:
This just a nit, but there's no need for the outer parentheses:
```
#if defined(__Fuchsia__)
```

Reply:
Done